src/Language/AST/NodeList.php

```diff
@@ -14,13 +14,16 @@
 use function count;
 use function is_array;
 
+/**
+ * @template T
+ */
 class NodeList implements ArrayAccess, IteratorAggregate, Countable
 {
-    /** @var Node[]|mixed[] */
+    /** @var T[] */
```

Comment:
Same here, let's keep `@var` for IDE support and use `@phpstan-var` for generic type

Reply:
So duplicate the types?